### PR TITLE
Disable metrics as a Protractor test

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

**Comment by mraible** [comment]
Test run happening at https://github.com/hipster-labs/jhipster-daily-builds/actions/runs/519460070

File: generators/client/templates/angular/src/test/javascript/e2e/admin/administration.spec.ts.ejs [hunk 1]
```diff
@@ -63,9 +63,9 @@ describe('administration', () => {
   });
   <%_ } _%>
 
+  <!-- This test tends to fail with Protractor, so we've disabled it. Please enable if it passes for you!
```
Comment (pascalgrimaud) [inline generators/client/templates/angular/src/test/javascript/e2e/admin/administration.spec.ts.ejs:66]: the comment is not correct, as it should be `//` or `/*` instead

Comment (mraible) [reply generators/client/templates/angular/src/test/javascript/e2e/admin/administration.spec.ts.ejs:66]: Doh!

**Comment by mraible** [comment]
Test run passed! ✅ 🕺

**Pull request merged**